### PR TITLE
Updates/fixes around network recovery

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/MessageList/NetworkDownIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useTranslationContext } from '../../contexts/translationContext/TranslationContext';
+import { styled } from '../../styles/styledComponents';
+
+const ErrorNotification = styled.View`
```
Comment (virdesai) [inline src/components/MessageList/NetworkDownIndicator.tsx:5]: 🤮  seeing styled-components again

**Review by virdesai — APPROVED** [review]
gonna need to make this change to v2 stuff as well?

File: src/components/MessageList/NetworkDownIndicator.tsx (hunk 1 above)
Comment (vishalnarkhede) [reply src/components/MessageList/NetworkDownIndicator.tsx:5]: hahaha

File: src/components/ChannelList/ChannelList.tsx [hunk 2]
```diff
@@ -306,6 +306,11 @@ export const ChannelList = <
     sort,
   });
 
+  useEffect(() => {
+    // TODO: Use sync api endpoint to refresh the list, instead of caliing queryChannels.
```
[the remark this answers is not in the document]
Comment (vishalnarkhede) [reply src/components/ChannelList/ChannelList.tsx:310]: We have an api to fetch all the missed events between two timestamps.

**Pull request merged**